### ISSUE TITLE
[FEAT] Rename Disks

### ISSUE THREAD

**Issue by Mike710Shine** [AnalogJ/scrutiny#531]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to be able to name the Disks because i have 5 Data Disks and 1 Paraty and when i look in scrutiny i have to look ot sdg or sda or so to find the right disk. It would be eseaser when i can name the disks.

**Additional context**
For example screenshot edited.
![grafik](https://github.com/AnalogJ/scrutiny/assets/19763591/65bc5c8d-595b-40ca-8282-e685dab96699)

**Comment by HotBlack77** [comment]
This would be really useful. 
Could it be as simple as letting users add a list of UUIDs and names in the config file?

**Comment by AnalogJ** [comment]
Scrutiny already collects your device partition labels. Have you tried changing the `Display title` setting?

<img width="645" alt="Screen Shot 2024-01-24 at 6 32 05 AM" src="https://github.com/AnalogJ/scrutiny/assets/891875/2f7ff0b1-4def-4e57-8ede-b1e5f16bd3e9">

**Comment by Mike710Shine** [comment]
Yes i tried it. But more than SDF i didn't get. Because i didn't have set an patition name. I would recommend it to change the name of each Harddrive. This would be nice.

**Comment by HotBlack77** [comment]
I get the following using "display title" option in settings (the one you showed in your screenshot). 
 - Name = device path, e.g. /dev/sdX (although its styled in all-caps) 
 - Serial ID = device serial, e.g. /BY-ID/ATA-TOSHIBA_HDWQ140_xxxxx
 - UUID = Same as for Name option, that is device path
 - Label = Same as for Name option, that is device path

I am running master#a3dfce3 in a docker container using the compose file you suggest.

**Comment by xHyperElectric** [comment]
This is my most desired feature

**Comment by sherbibv** [comment]
I too am interested in this feature. I have multiple drives with the same name/UUID/Label with the current implementation and I can't tell them appart. Being able to add a label in the webui would be great

**Comment by AnalogJ** [comment]
I think we can support overriding the disk label in the collector config file by making changes to https://github.com/AnalogJ/scrutiny/blob/master/collector/pkg/models/scan_override.go

and then wiring it up to the Label field in the device. Anyone have some time to take a look and open a PR?

**Comment by Eddy555** [comment]
I've found on my system any disk with a partition does not show the label
i.e. the following just shows /dev/sda
/dev/sda
/dev/sda1  Label = Disk 1

However I have 1 disk that is
/dev/sdb Label = Disk 2
and this works fine,

**Comment by robflate** [comment]
Has any progress been made on this? Thanks.

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.

**Comment by Eddy555** [comment]
Hi, this is still relevant for me, as outlined in my earlier comment. Thanks
